rules/windows/defense_evasion_workfolders_control_execution.toml

```diff
@@ -0,0 +1,77 @@
+[metadata]
+creation_date = "2022/03/02"
+maturity = "production"
+updated_date = "2022/03/02"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identifies the use of Windows Work Folders to execute a potentially masqueraded control.exe file in the current working directory. Misuse of Windows Work Folders could indicate malicious activity. 
```

Comment:
can you lint this with `python -m detection_rules toml-lint -f <this-file> or manually line break it as 120 chars